src/ui/handlers/message-ui-handler.ts

```diff
@@ -36,7 +36,18 @@ export abstract class MessageUiHandler extends AwaitableUiHandler {
     }
   }
 
-  showText(
+  /**
+   * Displays a dialogue message on the UI with optional delay and speaker name.
+   *
+   * @param text - The dialogue message to display.
+   * @param _name - (Optional) The name of the speaker for the dialogue. Defaults to an empty string if not provided.
+   * @param delay - (Optional) The delay in milliseconds before the dialogue is displayed. Defaults to `null` for no delay.
+   * @param callback - (Optional) A callback function to execute after the dialogue is displayed. Defaults to `null` for no callback.
+   * @param callbackDelay - (Optional) The delay in milliseconds before executing the callback. Defaults to `null` for no delay.
+   * @param prompt - (Optional) Whether to show the prompt after displaying the dialogue.
+   * @param promptDelay - (Optional) The delay in milliseconds before showing the prompt. Defaults to `null` for no delay.
```

Comment:
I wonder why Co-Pilot added the `_name` thing.
Can't trust these AI things.
Sorry for the inconvenience, should have checked more
```suggestion
   * Displays a dialogue message on the UI with optional delay
   *
   * @param text - The dialogue message to display.
   * @param delay - (Optional) The delay in milliseconds before the dialogue is displayed. Defaults to `null` for no delay.
   * @param callback - (Optional) A callback function to execute after the dialogue is displayed. Defaults to `null` for no callback.
   * @param callbackDelay - (Optional) The delay in milliseconds before executing the callback. Defaults to `null` for no delay.
   * @param prompt - (Optional) Whether to show the prompt after displaying the dialogue.
   * @param promptDelay - (Optional) The delay in milliseconds before showing the prompt. Defaults to `null` for no delay.
```

Reply:
I should've too, got a bit lazy on that one :')